packages/worker/src/internals/NoopWorkerProperty.ts

```diff
@@ -25,4 +25,10 @@ export class NoopWorkerProperty<Ts> implements WorkerProperty<Ts> {
   run(): Promise<PreconditionFailure | PropertyFailure | null> {
     throw new Error('Method not implemented.');
   }
+  runBeforeEach(): Promise<void> {
```

Comment:
Should be done in a separate PR

Reply:
Done in #4467

Let's consider this change as a no change from a versioning point-of-view.